WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -249,7 +249,13 @@ public class NotificationSettingsViewController : UIViewController
             cell.textLabel?.text            = settings.blog?.blogName ?? settings.channel.description()
             cell.detailTextLabel?.text      = settings.blog?.displayURL ?? String()
             cell.accessoryType              = .DisclosureIndicator
-            cell.imageView?.setImageWithSiteIcon(settings.blog?.icon)
+            
+            if let siteIconURL = settings.blog?.icon {
+                cell.imageView?.setImageWithSiteIcon(settings.blog?.icon)
```

Comment:
Ohhh snap! not sure how this happened, will get it fixed shortly. Thanks @sendhil!